playbooks/cnf/deploy-cnf-config.yaml

```diff
@@ -223,7 +223,7 @@
         state: present
         definition: "{{ lookup('template', 'templates/sctp_load_mc.j2') }}"
 
-    - name: Wait for MachineConfigPool update
+    - name: Wait for MachineConfigPool update (again)
```

Comment:
please rephrase this

Reply:
@eifrach , do mean a more descriptive rephrasing?